mne/io/edf/edf.py

```diff
@@ -204,8 +204,8 @@ def _read_segment_file(self, data, idx, fi, start, stop, cals, mult):
                     # make sure events without duration get one sample
                     n_stop = n_stop if n_stop > n_start else n_start + 1
                     if any(stim[n_start:n_stop]):
-                        raise NotImplementedError('EDF+ with overlapping '
-                                                  'events not supported.')
+                        warn('EDF+ with overlapping events'
+                             ' are not fully supported')
```

Comment:
something like?
```py
raw_fnames = eegbci.load_data(subject=88, runs)
with warnings.catch_warnings(record=True):
    raw_files = [read_raw_edf(f, preload=True) for f in raw_fnames]
assert_equal(len(w), len(runs))
```
just to make sure we can indeed read files with overlapping events and that it throws the correct warnings.